.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+## Bug Report
+<!--
+    If you are not submitting a bug report, ignore this template and see
+      http://Chapel-Issues-Best-Practices.com  (TODO)
+  -->
+### Summary of Problem
+
+
+### Steps to Reproduce
+
+**What behavior do you observe when encountering this issue?**
+
+**Please provide one of the following:**
+
+- Reduced program demonstrating issue *(preferred)*:
+- Original program demonstrating issue:
+- Description of how to recreate the issue *(last resort)*:
+
+**Please provide any of the following that are necessary to reproduce the issue:**
+
+- How to compile:
+- How to run:
+- Input files:
+- Helper modules:
```

Comment:
I might move "Helper modules" to the top of this list since it's a pre-requisite to compiling.  Alternatively, we could remove it with the understanding that it's part of the source code you'd need to provide (that said, it isn't uncommon for people to send us a file by email and forget to include the helper modules which is why this is called out to begin with).